react/FilterNavigator.js

```diff
@@ -159,6 +164,14 @@ const FilterNavigator = ({
     (facet) => facet.selected
   )
 
+  const { searchQuery } = useSearchPage()
+  const hasFiltersApplied = searchQuery.variables.selectedFacets.length > 1
+
+  const { history, navigate } = useRuntime()
+  const handleResetFilters = () => {
+    navigate({ to: history.location.pathname })
```

Comment:
This function removes any querystring from the URL and this is a problem. There are some querystrings  that still needed after cleaning the filters

![Ac5lOt2Om6](https://user-images.githubusercontent.com/40380674/123177722-323efa80-d45c-11eb-9c90-d41f09cd992e.gif)

In this case for example. The proper URL should be:

 https://hiago--storecomponents.myvtex.com/top?_q=top&fuzzy=0&map=ft&operator=and&query=/top&searchState

But the button is redirecting to:

https://hiago--storecomponents.myvtex.com/top

In other words, this function is removing any querystring from the URL, even querystrings that are not related to the filters at all

I would recommend using the `navigateToFacet` function. This function will do all the work of setting the proper URL. You just need to pass all the selected facets. I think it is gonna solve the problem

Reply:
Hello, How can i use that function? I can't figured it out how to use it

Reply:
the function is decleared here: https://github.com/vtex-apps/search-result/blob/6d1e7666c07ad5d521c553c41dccceb78f3c632d/react/FilterNavigator.js#L163

and here is how we use it to clean the filters on mobile: 
https://github.com/vtex-apps/search-result/blob/ce253793d668ff84af8bb47330ff7c9ddd1c8271/react/components/FilterSidebar.js#L143

Reply:
https://clearall--mcxhoreca.myvtex.com/brutarie-si-patiserie?_q=Brutarie-si-patiserie&map=category-1,brand,temperatura-minima,cantitate-pachet&query=/brutarie-si-patiserie/la-lorraine/-18/40&searchState